app/src/main/res/layout/component_locale_settings.xml

```diff
@@ -10,7 +10,6 @@
         android:id="@+id/locale_list"
         android:layout_width="match_parent"
         android:layout_height="0dp"
-        android:layout_margin="@dimen/locale_list_margin"
```

Comment:
Since the margin is removed here, perhaps an end margin of 16dp should be added to the list items?

Reply:
Do we really need end margin? Why not to use full width and length?

Reply:
If the string is really long, it will hit the end of the screen rather than having a little bit of spacing.

Reply:
Hmm you're right, I'll add margin end to list item.

Reply:
Fixed.

Reply:
@NotWoods I've added 16dp margin end but I'm a little confused.
In the ux mock (I've shared ss above) there is 32dp margin end on "Follow device language" and 48dp margin end on others.
Which one it should be or left is as is?

Reply:
I'm not sure, cc @brampitoyo

Reply:
@hakkikaancaliskan @NotWoods Sorry, my mockup contained an error. The end margin should be 16dp, not 32.

In fact, for every list item that doesn’t have an icon or a toggle to the right, the end margin should always be 16dp. If there’s an icon or a toggle, it would of course be different (because otherwise it would run over the icon/toggle element).

Can we make sure that we follow that in other list items?

Reply:
@NotWoods currently all items have 16dp end margin. Ready to land?